### ISSUE TITLE
test on raspi result 

### ISSUE THREAD

**Issue by shartoo** [duangenquan/YoloV2NCS#31]
This repo can run on raspi 3 b+ successful, demo test (yolov2-tiny) time cost 715ms.

**Comment by akde** [comment]
@shartoo thx for sharing the results. According to [this repo](https://github.com/shizukachan/darknet-nnpack) you can achieve 1.3 fps (similar to yours) using yolov3-tiny WITHOUT the ncs. I followed the instructions and obtained the same results.

@duangenquan So is it possible to combine ncs with [ninja](https://ninja-build.org/) and [NNPACK](https://github.com/shizukachan/NNPACK) to get even higher FPS?

**Comment by shartoo** [comment]
@akde Yes i did test another [darknet-nnpack](https://github.com/digitalbrain79/darknet-nnpack) on raspi for yolov2 ,coco cfg with 8 secs per image.But i want to detect with yolov3 ,while that repo failed with fatal error.

**Comment by akde** [comment]
@shartoo does it have to be on RP? I am also trying to implement yolov3 on a SBC. I will try udoo X86 Advanced plus. Have you ever tried it on another SBC?

**Comment by shartoo** [comment]
Actually the author of [darknet-nnpack](https://github.com/digitalbrain79/darknet-nnpack) had implemented yolov3 branch  but it did't work.You can do a RP,my suggestion is implement it with another darknet [AlexeyAB darknet](https://github.com/AlexeyAB/darknet) other than original [pjreddie darknet](https://github.com/pjreddie/darknet) . I'm not familiar to c++/c ,good lock to you!
